easycord/plugins/_shared.py

```
@@ -0,0 +1,46 @@
+"""Shared helpers for the bundled EasyCord plugins."""
+from __future__ import annotations
+
+import json
+import os
+from pathlib import Path
+
+import discord
+
+
+def require_guild(ctx):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `require_guild`

(ANN202)